src/net.cpp

```diff
@@ -3216,6 +3225,29 @@ void CNode::DisconnectIfBanned()
     }
 }
 
+void CNode::UpdateNodeConfig()
```

Comment:
nit: I find the name of this function "confusing", since this function keeps track of the network polices exposed by a peer via xversion why not name it `UpdatePeerConfig`?

Reply:
I named it that because within its context of a CNode member function "Node" refers to the CNode, and Peer would refer to a different peer this CNode has.  But I can see how within the larger context Node refers to this node.  How about ReadConfigFromXVersion()?

Reply:
`ReadConfigFromXVersion()` is fine by me